modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/socket/TransitionService.java

```diff
@@ -162,7 +164,13 @@ private void registerListeners() {
 
         ignite.message().localListen(SEND_RESPONSE, new MessagingListenActor<WebSocketEvent>() {
             @Override protected void receive(UUID nodeId, WebSocketEvent evt) {
-                browsersSrvc.sendResponseToBrowser(evt);
+                boolean processed = browsersSrvc.processResponse(evt);
+
+                if (!processed)
+                    processed = agentsSrvc.processResponse(evt);
+
+                if (!processed)
```

Comment:
Why we again check this condition?

Reply:
We check condition to show warning that response was not processed.